hw/ip/rstmgr/doc/dv/index.md

```diff
@@ -122,6 +130,4 @@ $ $REPO_TOP/util/dvsim/dvsim.py $REPO_TOP/hw/ip/rstmgr/dv/rstmgr_sim_cfg.hjson -
 ```
 
 ## Testplan
-<!-- TODO: uncomment the line below after adding the testplan.
-Please make sure the testplan is added to `/util/build_docs.py`. -->
-{{</* incGenFromIpDesc "hw/ip/rstmgr/data/rstmgr_testplan.hjson" "testplan" */>}}
+{{< incGenFromIpDesc "../../data/rstmgr_testplan.hjson" "testplan" >}}
```

Comment:
Just curious - any reason we are not keeping the full path anymore? Is that because it is under top_earlgrey/ip/...

Reply:
All index.md files use relative paths for this. However, this also works, but notice the initial "/".
{{< incGenFromIpDesc "/hw/ip/rstmgr/data/rstmgr_testplan.hjson" "testplan" >}}

Reply:
It seems if uvmdvgen suggested an initial "/" we would end up without ".."'s in the path, which seems better.